.node-version

```diff
@@ -0,0 +1 @@
+7.5.0
```

Comment:
Should we drop the engine version from package.json to avoid synchronicity issues? Also, please consider updating the development set up notes in the readme!

Reply:
Yeah it would be ideal to only have to update this in one place. Does anything else depend on the engine version?

Reply:
I think you get a warning if you try to install with an older version of Node but that's it.

https://docs.npmjs.com/files/package.json#engines